### ISSUE TITLE
Catalogue as an aspect

### ISSUE THREAD

**Issue by fnielsen** [fnielsen/scholia#327]
For instance, for https://www.wikidata.org/wiki/Q51456640

**Comment by fnielsen** [comment]
```
SELECT
  ?code
  ?item ?itemLabel
  ?type
WHERE {
  ?item p:P972 [
    ps:P972 wd:Q51467536 ;
    pq:P528 | pq:P1545 ?code
  ] .
  
  OPTIONAL { ?item wdt:P31 / rdfs:label ?type . FILTER (LANG(?type) = 'en') }
  
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
}
ORDER BY ?code
```

**Comment by fnielsen** [comment]
Implemented with 0e8bf3e37e847bf64b84f3d52faba0a65035e609
